hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/schema/UtilizationSchemaDefinition.java

```diff
@@ -52,8 +59,12 @@
   @Transactional
   public void initializeSchema() throws SQLException {
     Connection conn = dataSource.getConnection();
-    createClusterGrowthTable(conn);
-    createFileSizeCount(conn);
+    if (!TABLE_EXISTS_CHECK.test(conn, FILE_COUNT_BY_SIZE_TABLE_NAME)) {
+      createFileSizeCount(conn);
```

Comment:
Can we rename to createFileSizeCountTable for consistency?

Reply:
Done.